contentcuration/contentcuration/frontend/shared/views/details/Details.vue

```diff
@@ -218,28 +262,40 @@
       LoadingText,
       ContentNodeIcon,
       ExpandableList,
+      CopyToken,
       DetailsRow,
       Thumbnail,
     },
     mixins: [fileSizeMixin, constantsTranslationMixin],
     props: {
-      nodeID: {
-        type: String,
+      details: {
```

Comment:
Could you add a comment clarifying what this is?

Reply:
Yup